config/app.go

```diff
@@ -54,6 +54,8 @@ const (
 	SlackDeployUsernames    = "slack-deploy-usernames"
 	RolloutTimeout          = "rollout-timeout"
 	JobRunTimeout           = "job-run-timeout"
+	CiProvider              = "ci-provider"
+	CircleciToken           = "circleci-token"
```

Comment:
This should probably be named `CircleCIToken`

Reply:
Done